toqito/channels/tests/test_partial_trace.py

```diff
@@ -35,345 +47,582 @@ def test_partial_trace_cvxpy():
+                    [9610, 9626, 9642, 9658, 9674, 9690, 9706, 9722, 9738, 9754, 9770, 9786, 9802, 9818, 9834, 9850],
+                    [
+                        10634,
+                        10650,
+                        10666,
+                        10682,
+                        10698,
+                        10714,
+                        10730,
+                        10746,
+                        10762,
+                        10778,
+                        10794,
+                        10810,
+                        10826,
+                        10842,
+                        10858,
+                        10874,
+                    ],
+                    [
+                        11658,
+                        11674,
+                        11690,
+                        11706,
+                        11722,
+                        11738,
+                        11754,
+                        11770,
+                        11786,
+                        11802,
+                        11818,
+                        11834,
+                        11850,
+                        11866,
+                        11882,
+                        11898,
+                    ],
+                    [
+                        12682,
+                        12698,
+                        12714,
+                        12730,
+                        12746,
+                        12762,
+                        12778,
+                        12794,
+                        12810,
+                        12826,
+                        12842,
+                        12858,
+                        12874,
+                        12890,
+                        12906,
+                        12922,
+                    ],
+                    [
+                        13706,
+                        13722,
+                        13738,
+                        13754,
+                        13770,
+                        13786,
+                        13802,
+                        13818,
+                        13834,
+                        13850,
+                        13866,
+                        13882,
+                        13898,
+                        13914,
+                        13930,
+                        13946,
+                    ],
+                    [
+                        14730,
+                        14746,
+                        14762,
+                        14778,
+                        14794,
+                        14810,
+                        14826,
+                        14842,
+                        14858,
+                        14874,
+                        14890,
+                        14906,
+                        14922,
+                        14938,
+                        14954,
+                        14970,
+                    ],
+                    [
+                        15754,
```

Comment:
I am not too sure if we want to use `black`. 

yes, it will make sure all lines are of length 120. it also makes these weird formatting changes for numpy arrays.